ReleaseNotes.md

```diff
@@ -2,6 +2,7 @@
 -----------------------
 ### December 2018
 - ODE-466 serialId element of metadata will now provide a true serial number to help identify missing records.
+- ODE-964 Added rsuUsername/rsuPassword to application.properties and ODE_RSU_USERNAME and ODE_RSU_PASSWORD values to sample.env
```

Comment:
Release notes should be informing the user. Please change to: `Added ode.rsuUsername/ode.rsuPassword  application properties or alternatively ODE_RSU_USERNAME and ODE_RSU_PASSWORD environment variables to authenticate communications with RSU instead of using the REST API parameters.`